CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ find_package(catkin REQUIRED
   std_srvs
   tf2_ros
   tf2_geometry_msgs
+  tf2_eigen
```

Comment:
This also needs to be added to package.xml as a `build_depend`.

Reply:
Done